source/html/implementation/vss-xml-implementation-html_writer_data.ads

```diff
@@ -0,0 +1,459 @@
+--
+--  Copyright (C) 2022, AdaCore
+--
+--  SPDX-License-Identifier: Apache-2.0
+--
+
+with VSS.Strings;
+
+package VSS.XML.Implementation.HTML_Writer_Data is
+
+   --  pragma Preelaborate;
+
+   --  HTML elements are reordered to group together all elements that appear
+   --  in the tag omitting conditions.
```

Comment:
IMHO, it would be helpful to have a link to the document, source of the corresponding information.